ui/widgets/sortable.js

```diff
@@ -231,11 +236,6 @@ return $.widget( "ui.sortable", $.ui.mouse, {
 			relative: this._getRelativeOffset()
```

Comment:
yeah my bad. not sure how the relative position works since we're always setting it to absolute. but i've moved it around so relative offset is set before the position is changed to absolute.